tests/js/client/shell/shell-arangosearch-dangling-link-cluster.js

```diff
@@ -0,0 +1,95 @@
+/*jshint globalstrict:false, strict:false, maxlen : 4000 */
+/* global arango, assertTrue, assertEqual */
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2023 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Andrei Lobov
+////////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+const jsunity = require('jsunity');
+const db = require("@arangodb").db;
+const internal = require('internal');
+
+
+function ArangoSearchDanglingLinkSuite () {
+  return {
+    setUpAll : function () {
+      internal.debugClearFailAt();
+      db._useDatabase('_system');
+      db._createDatabase('UnitTestsDangling');
+      db._useDatabase('UnitTestsDangling');
+      db._create("foo");
+      db.foo.save({a:1, b:'text'});
+    },
+    
+    setUp : function() {
+        internal.debugClearFailAt();
```

Comment:
Slightly inconsistent indentation
```suggestion
      internal.debugClearFailAt();
```